### PR TITLE
Add timeframe warning to readme

### PR DESCRIPTION
**Pull request by dichotommy**
Add warning for contributors that PRs which haven't been updated by the author in 7+ days will be closed.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -6,13 +6,21 @@ This repository is the documentation for the **open-source search engine API [Me
 
 ## Contributing
 
-Hey, thank you so much for wanting to contribute! If you'd like to help improve this documentation, we'd love to have you! If you think something is missing or could be improved, issues are highly appreciated.
+Hey, thank you so much for wanting to contribute! If you'd like to help improve this documentation, we'd love to have you!
 
-Before contributing, please leave a comment on an [issue](https://github.com/meilisearch/documentation/issues) letting us know you're interested. Only submit a Pull Request after you have been assigned a corresponding issue.
+Before opening a PR, please look through our [open issues](https://github.com/meilisearch/documentation/issues) to see if one already exists for the problem you'd like to solve. If yes, please leave a comment letting us know you're interested. If no, please open a new issue describing the problem and telling us you want to work on it.
```
Comment (dichotommy) [inline README.markdown:11]:
```suggestion
Before opening an issue or PR, please look through our [open issues](https://github.com/meilisearch/documentation/issues) to see if one already exists for your problem. If yes, please leave a comment letting us know that you're waiting for a fix or willing to work on it yourself. If not, please open a new issue describing the problem and informing us whether you want to work on it or not.
```

File: README.markdown [hunk 2]
```diff
@@ -6,13 +6,21 @@ This repository is the documentation for the **open-source search engine API [Me
...
+Hey, thank you so much for wanting to contribute! If you'd like to help improve this documentation, we'd love to have you!
 
-Before contributing, please leave a comment on an [issue](https://github.com/meilisearch/documentation/issues) letting us know you're interested. Only submit a Pull Request after you have been assigned a corresponding issue.
+Before opening an issue or PR, please look through our [open issues](https://github.com/meilisearch/documentation/issues) to see if one already exists for your problem. If yes, please leave a comment letting us know that you're waiting for a fix or willing to work on it yourself. If not, please open a new issue describing the problem and informing us whether you want to work on it or not.
 
-See something wrong that's not on the list? Great! Open a new issue describing the problem and tell us you want to work on it.
+To contribute, clone this repository, commit your changes and open a pull request. [Read more about working on the docs site locally.](#local-development)
```
Comment (dichotommy) [inline README.markdown:13]:
```suggestion
To contribute, clone this repository, commit your changes and open a pull request. [Read more about working on this site locally.](#local-development)
```

**Comment by guimachiavelli** [comment]
Do you think we should also alter our PR message template with a (summarised) version of that?

**Review by maryamsulemani97 — APPROVED** [review]
🦇

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**